pkg/codegen/python/gen.go

```diff
@@ -539,28 +539,28 @@ _package_lock = asyncio.Lock()
 _package_ref = ...
 async def get_package():
 	global _package_ref
-	# TODO: This should check a feature flag for if RegisterPackage is supported.
 	if _package_ref is ...:
-		async with _package_lock:
-			if _package_ref is ...:
-				monitor = pulumi.runtime.settings.get_monitor()
-				parameterization = resource_pb2.Parameterization(
-					name=%q,
-					version=get_version(),
-					value=base64.b64decode(%q),
-				)
-				registerPackageResponse = monitor.RegisterPackage(
-					resource_pb2.RegisterPackageRequest(
+		if pulumi.runtime.settings._sync_monitor_supports_package_references():
```

Comment:
I think you want to rename this to `_sync_monitor_supports_parameterization`

Reply:
I thought I did! git foo failing me